.travis.yml

```diff
@@ -0,0 +1,11 @@
+# This is experimental
```

Comment:
Travis' support of C# is experimental, so it might be worth it to indicate it here also.